packages/embark/src/lib/modules/blockchain_listener/index.js

```diff
@@ -48,47 +56,64 @@ class BlockchainListener {
 
   _registerConsoleCommands() {
     this.embark.registerConsoleCommand({
-      description: 'Toggles regular transactions used to prevent transactions from getting stuck when using Geth and Metamask',
-      matches: ['regularTxs on', 'regularTxs off'],
-      usage: "regularTxs on/off",
+      description: __('Toggles regular transactions used to prevent transactions from getting stuck when using Geth and Metamask'),
+      matches: ['devtxs on', 'devtxs off', 'regularTxs on', 'regularTxs off'],
+      usage: "devtxs on/off",
       process: (cmd, callback) => {
-        const eventCmd = `regularTxs:${cmd.trim().endsWith('on') ? 'start' : 'stop'}`;
-        this.events.request(eventCmd, callback);
+        if (cmd.startsWith('regularTxs')) {
+          this.logger.info(__("Deprecation notice: The command 'regularTxs on/off' is now deprecated in favor of 'devtxs on/off' and will be removed in future versions."));
+        }
+        const enable = cmd.trim().endsWith('on');
+        this.logger.info(`${enable ? 'Enabling' :  'Disabling'} regular transactions...`);
+        if(enable) {
+          return this._startRegularTxs(() => {
+            const message = __('Regular transactions have been enabled');
+            this.logger.info(message);
+            callback(null, message);
```

Comment:
Won't this log the message twice?

Reply:
Oddly, the normal console log only shows the message the logger is given, and the cockpit console and embark console (`embark console`) only show what is returned in the command callback.

Reply:
Ah ok, weird but fine